src/Sylius/Bundle/CoreBundle/Resources/translations/validators.de_CH.yml

```diff
@@ -100,8 +100,6 @@ sylius:
     review:
         author:
             not_blank: Bitte geben Sie Ihre E-Mail-Adresse ein.
-        rating:
-            range: Bewertung muss eine Ganzzahl im Bereich 1-5 sein.
```

Comment:
I guess it should be removed from all locales

Reply:
it's already removed from all locales, just a few of them had that translation defined.

Reply:
```bash
$ cd src/Sylius/Bundle/CoreBundle/Resources/translations 
$ ag -i range
validators.en.yml
117:            not_in_range: Review rating must be between {{ min }} and {{ max }}.

```